src/schema/partner_show_event.ts

```diff
@@ -1,8 +1,46 @@
-import date from "./fields/date"
-import { GraphQLString, GraphQLObjectType } from "graphql"
+import dateField, { date, DateSource } from "./fields/date"
+import { GraphQLString, GraphQLObjectType, GraphQLFieldConfig } from "graphql"
 import { ResolverContext } from "types/graphql"
 import { exhibitionPeriod } from "lib/date"
 
+const hasOldEmissionUserAgentString = (userAgent): boolean =>
```

Comment:
Btw, you should always type the input parameters, otherwise TS defaults to `any`.

Reply:
Thanks, fixed (think I missed this bc the feedback about it in VSC is so understated).

Reply:
This is one of those more strict options that we haven’t been using yet. There’s a compiler option that disallows any implicit `any` and there’s a TSLint rule to require parameter types.